src/main/js/controls/Line/Line.js

```diff
@@ -184,6 +188,47 @@ class Line extends GraphContent {
         return this;
     }
 
+    /**
+     * @inheritdoc
+     */
+    reflow(graph, graphData) {
+        this.config.values = graphData.values;
+        this.dataTarget = processDataPoints(graph.config, this.config);
+        const lineSVG = graph.svg
+            .select(`g[aria-describedby="${graphData.key}"]`)
+            .selectAll(`.${styles.line}`)
+            .data([this.dataTarget]);
+        drawDataLines(graph.scale, graph.config, lineSVG.enter());
+        lineSVG.exit().remove();
```

Comment:
Is this required ? what is the lineSVG object basically contain ?

Reply:
It is being used to update the lines, joining the datapoint.